src/main/webapp/resources/js/pages/projects/share/ShareLarge.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Row, Col, Alert, Progress } from "antd";
+import { shareSamplesWithProject } from "../../../apis/projects/samples";
+
```

Comment:
Add file comment

Reply:
Updated in 3d33ef3a2c24c2e41f437e89959bbeae2bf81cd2